tests/shopify_python/test_git_utils.py

```diff
@@ -248,3 +248,67 @@ def test_autopep_files(tmpdir):
     for fixed_file in files_to_autopep:
         with open(fixed_file) as file_to_check:
             assert file_to_check.readlines() == file_lines
+
+
+def test_linter(tmpdir):
+    # type: ('py.path.LocalPath') -> None
+    open(str(tmpdir.join('__init__.py')), 'w')
+
+    file_lines = [
+        "def foo():",
+        "  return 1\n"
+    ]
+    tmpdir.join('file.py').write('\n'.join(file_lines))
```

Comment:
Talked IRL. Since what we're aiming to test is whitespace, alright, this syntax is justified.